packages/snippets/pubspec.yaml

```diff
@@ -2,21 +2,21 @@ name: snippets
 description: A package for parsing and manipulating code samples in Flutter repo dartdoc comments.
 repository: https://github.com/flutter/assets-for-api-docs/tree/master/packages/snippets
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+snippets%22
-version: 0.3.0
+version: 0.3.1
 
 environment:
-  sdk: ">=2.17.0-0 <3.0.0"
+  sdk: '>=3.0.0 <4.0.0'
 
 dependencies:
-  analyzer: 4.3.1
+  analyzer: ^5.13.0
   args: ^2.0.0
-  dart_style: ^2.2.3
-  file: ^6.1.0
+  dart_style: ^2.3.1
+  file: ^6.0.0-nullsafety.4
```

Comment:
Actually, looks like I was wrong about this one: let's keep `^6.1.0`, which also appears to work.

Reply:
`file: ^6.1.4` is the newer one, do we still want to keep `6.1.0`?

Reply:
If `^6.1.4` works, then sure.  The latest version of `file` is `7.0.0`, which won't work.

Reply:
`^6.1.4` seems to fail so I just reverted to `6.1.0` fingers crossed :)